Sources/StreamChatUI/ChatMessageList/ChatMessageListTitleView.swift

```diff
@@ -5,44 +5,60 @@
 import StreamChat
 import UIKit
 
-// TODO: Audit
-public class ChatMessageListTitleView<ExtraData: ExtraDataTypes>: UIView, UIConfigProvider {
-    public var title: String? {
-        get { titleLabel.text }
-        set { titleLabel.text = newValue }
+public typealias ChatMessageListTitleView = _ChatMessageListTitleView<NoExtraData>
+
+/// A view that is used as a wrapper for status data in navigationItem's titleView
+open class _ChatMessageListTitleView<ExtraData: ExtraDataTypes>: _View, UIConfigProvider {
+    /// Content of the view that contains title (first line) and subtitle (second nil)
+    open var content: (title: String?, subtitle: String?) = (nil, nil) {
+        didSet {
+            updateContentIfNeeded()
+        }
     }
     
-    public var subtitle: String? {
-        get { subtitleLabel.text }
-        set { subtitleLabel.text = newValue }
-    }
+    /// Label that represents the first line of the final titleView
+    open private(set) lazy var titleLabel: UILabel = UILabel()
+    
+    /// Label that represents the second line of the final titleView
+    open private(set) lazy var subtitleLabel: UILabel = UILabel()
     
-    private weak var titleLabel: UILabel!
-    private weak var subtitleLabel: UILabel!
+    /// A view that acts as the main container for the subviews
+    open private(set) lazy var containerView: ContainerStackView =
+        ContainerStackView(
+            axis: .vertical,
+            alignment: .center,
+            spacing: 0,
+            views: []
+        )
```

Comment:
Ideally, all these things should be set up in `setUpLayout` rather than in the initializer.